04 StringCombinations/04-StringCombinations.html

```diff
@@ -0,0 +1,31 @@
+<!DOCTYPE html>
+<html>
+  <head>
+    <meta charset="utf-8">\
+    <script src = "https://ajax.googleapis.com/ajax/libs/jquery/3.1.1/jquery.min.js"></script>
+  </head>
+  <body>
+    <input id= "string"></input>
+    <button onclick="combo()">Create</button>
+    <div id = "show"></div>
+
+    <script>
+    let combo = () => {
+    let inp = document.getElementById("string").value;
```

Comment:
Let's make the variable names more useful. In this case I think you are trying to say `input`.